package.json

```diff
@@ -39,7 +39,7 @@
     "istanbul": "^0.4.5",
     "mocha": "^5.2.0",
     "typescript": "^3.1.3",
-    "typescript-eslint-parser": "^20.0.0"
+    "typescript-eslint-parser": "20.0.0"
```

Comment:
i think this is now fixed, so now we can use `^` again

Reply:
Right, but should we say `^20.1.1` now?

Reply:
yes, good call